src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -99,6 +133,27 @@ limitations under the License.
       <access op="read" privilege="view"/>
     </attribute>
 
+    <attribute side="server" code="0x0005" define="COMMISSIONING_ARL" type="array" entryType="CommissioningAccessRestrictionEntryStruct" writable="false" optional="true">
+      <description>CommissioningARL</description>
+      <access op="read" privilege="view"/>
+    </attribute>
+
+    <attribute side="server" code="0x0006" define="ARL" type="array" entryType="AccessRestrictionEntryStruct" writable="false" optional="true">
+      <description>ARL</description>
+      <access op="read" privilege="view"/>
+    </attribute>
+
+    <command code="0x00" source="client" name="ReviewFabricRestrictions" optional="true" response="ReviewFabricRestrictionsResponse">
+      <description>This command signals to the service associated with the device maker that the fabric administrator would like a review of the current fabric restrictions.</description>
+      <access op="invoke" privilege="administer"/>
+      <arg id="0" name="ARL" array="true" type="CommissioningAccessRestrictionEntryStruct" optional="true"/>
```

Comment:
```suggestion
      <arg id="0" name="ARL" array="true" type="CommissioningAccessRestrictionEntryStruct" optional="false"/>
```
Doesn't look optional in the spec PR to me....

Reply:
Done.